src/icalendar/prop.py

```diff
@@ -48,21 +48,21 @@
 from icalendar.caselessdict import CaselessDict
 from icalendar.parser import Parameters
 from icalendar.parser import escape_char
-from icalendar.parser import tzid_from_dt
 from icalendar.parser import unescape_char
 from icalendar.parser_tools import DEFAULT_ENCODING
 from icalendar.parser_tools import SEQUENCE_TYPES
 from icalendar.parser_tools import to_unicode
 from icalendar.parser_tools import from_unicode
-from icalendar.timezone_cache import _timezone_cache
-from icalendar.windows_to_olson import WINDOWS_TO_OLSON
 
 import base64
 import binascii
-import pytz
+from .timezone import tzp
```

Comment:
I'd not mix relative and absolute imports. Since we have more absolute imports, better use `from icalendar.timezone import tzp`